### PR TITLE
DRILL-5165: limitRel to return correct rows for limit all case

### PR DESCRIPTION
**Pull request by chunhui-shi**

### CODE REVIEW

File: exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/limit/TestLimitWithExchanges.java [hunk 1]
```diff
@@ -125,6 +125,15 @@ public void testLimitImpactExchange() throws Exception {
     }
   }
 
+  @Test
+  public void TestLimitAllOnParquet() throws Exception {
```
Comment (jinfengni) [inline exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/limit/TestLimitWithExchanges.java]: Have you tried this new testcase without the change above? Will it successful or fail?

**Comment by chunhui-shi** [comment]
It failed without the fix.

________________________________

File: exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/limit/TestLimitWithExchanges.java (hunk 1 above)
Comment (chunhui-shi) [inline exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/limit/TestLimitWithExchanges.java]: update the tested query. I did not notice the query in unit test is slightly different than the one I used in sqlline.

**Comment by jinfengni** [comment]
+1

**Pull request closed without merging**